deploy/chart/templates/20-packageserver.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: apiregistration.k8s.io/v1beta1
```

Comment:
In order for our packaging scripts to work we need each of these yamls to have its own file (threw them all in one for testing)

Alternately you could fix our packaging scripts to not require that each be in a separate file, whichever